### PR TITLE
Update 1906-12-17.xml

### PR DESCRIPTION
**Pull request by ParkerBryant**
Deliverable 2

### CODE REVIEW

**Comment by whanley** [comment]
Good progress intially. Maybe using a different OCR program might help? Please divide page 5 and so on into items with headers. Also, put all of the contents of each page within its own`<div type="page">` tags.

File: 1906-12-18.xml [hunk 1]
```diff
@@ -0,0 +1,7676 @@
...
+                            <cell cols="3">Wasta ... ARR.</cell>
+                            <cell>7.11 p.m.</cell>
+                            <cell>5.57 a.m.</cell>
+                        </row>
+                        <row>
+                            <cell cols="3">Luxor...ARR.</cell>
+                            <cell>11.35 p.m.</cell>
+                            <cell>10. 0 a.m.</cell>
+                            <cell cols="3">Cairo ... ARR.</cell>
+                            <cell
```
Comment (whanley) [inline 1906-12-18.xml]: `<div type="section" feature="local">`

Comment (whanley) [inline 1906-12-18.xml]: move this after the column breaks, and make it `<div type="section" feature="wire">`

Comment (whanley) [inline 1906-12-18.xml]: this div is unnecessary

Comment (whanley) [inline 1906-12-18.xml]: delete just this div tag, and the two paragraph tags around it--column break can come in the middle of a paragraph: `Meet-<cb n="3"/>ing, he must`

**Comment by whanley** [comment]
Generally really good, Parker. There are a few things to clean up. See my in-line comments above about your Tuesday issue. Do similar things with Monday. Pay particular attention to the feature attribute. If you run a `//@when` query, you'll find that there are some adverts and templates that you haven't updated. Almost complete!

File: 1906-12-20 [hunk 2]
```diff
@@ -0,0 +1,8561 @@
...
+                        <row>
+                            <cell cols="2"/>
+                            <cell>a.m.</cell>
+                            <cell>a.m.</cell>
+                            <cell>noon</cell>
+                            <cell>p.m.</cell>
+                            <cell>p.m.</cell>
+                            <cell>p.m.</cell>
+                            <cell>p.m.</cell>
+
```
Comment (huoyanzhizhou) [inline 1906-12-20]: local and general should be a section with a feature of "social", please check https://dig-eg-gaz.github.io/contents/features/ 
You have very good structures, but you may need to update your measures and content.

**Comment by whanley** [comment]
I'm going to merge these because there's lots of good content. Still lots of cleaning to do, however. Wednesday issue is empty, other than page 1.

**Pull request merged**